lightwood/mixer/lightgbm.py

```diff
@@ -127,6 +131,8 @@ def _to_dataset(self, data: Dict[str, Dict], output_dtype: str):
                     self.ordinal_encoder.fit(np.array(list(self.label_set)).reshape(-1, 1))
 
                 label_data = [x if x in self.label_set else '__mdb_unknown_cat' for x in label_data]
+                if weight_map is not None:
+                    data[subset_name]['weights'] = [weight_map[x] for x in label_data]
```

Comment:
weights are on columns-  wouldn't this be on rows?

Reply:
For Lightgbm weighting is on a per-row basis (there's only ever a single target column) ... only way to do a value-map is using the sklearn API, which just converts to this, and which is buggier